tests/helpers/server.ts

```diff
@@ -81,3 +81,9 @@ export const resetSetting = async (
   await page.getByRole("button", { name: "Close notification" }).click();
   await validateSettingValue(settingRow, defaultValue);
 };
+
+export const getInitialSettingValue = (page: Page, settingName: string) => {
```

Comment:
```suggestion
export const getServerSettingValue = (page: Page, settingName: string) => {
```
This function doesn't know about its return value being initial or update or modified :)